libyul/optimiser/DataFlowAnalyzer.cpp

```diff
@@ -223,15 +225,23 @@ void DataFlowAnalyzer::operator()(Block& _block)
 
 optional<YulString> DataFlowAnalyzer::storageValue(YulString _key) const
 {
-	if (YulString const* value = util::valueOrNullptr(m_state.environment.storage, _key))
+	if (YulString const* value = valueOrNullptr(m_state.environment.storage, _key))
```

Comment:
Probably makes sense to add something like ``valueOrNullopt`` to do just this as well (the main reason it's not there is that it has to copy, which may not always be desirable, but we can still have it for cases for which copying doesn't matter).

But no point in delaying this PR for that.

Reply:
Sounds like rust, then :D